src/atoms/typography/paragraph.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { forwardRef } from "react";
+
+// -------------------------------------------------------------------------------------------------
+// #region Enums
+// -------------------------------------------------------------------------------------------------
+
+export enum ParagraphSizes {
```

Comment:
Nitpick - consider breaking this enum/constant outside of the component

Reply:
Yeah, agree. Also a case where I need to change the value from baking in that it is a css modifier, which I've done other places. Instead, having the component take the 'value' and prepend `-` as needed. This allows for leveraging these enums and constants in other scenarios with JS/TS.